app/services/sessionsManager.js

```diff
@@ -5,10 +5,23 @@ const SECRET = config.common.session.secret;
 
 exports.HEADER_NAME = config.common.session.header_name;
 
+exports.blackList = [];
```

Comment:
The idea of saving a blacklist of users and time of invaldiation is good, but we have some problem.
Saving it in an memory-stored array can cause as some troubles, like any server restart would clean the whole blacklist and all the tokens that were invalidated are now active again and if you happen to work with more than 1 server, using some load balancer for example, the tokens would be only invalidated in one instance of the server and not in the others. So if we wan't to implement something like this we need to save them in the database.
On the other hand, you are making an array that then you are searching by user email and that should be stored in database, doesn't it sound a lot like some table you already have?